src/strand-grid/strand-grid.js

```diff
@@ -358,6 +366,12 @@
 		removeColumn: function (column) {
 			return Polymer.dom(this).removeChild(column);
 		},
+
+		_showNoResults(noResultsMessage, data) {
+			if (noResultsMessage && data) {
```

Comment:
something like `return noResultsMessage ? (!data || !data.length) : false;` feels right to me (because I'm not in love with the implicit `return undefined; // which just happens to be falsey` caused by this "else-less" if statement)